src/metabase/driver/common/parameters/dates.clj

```diff
@@ -110,43 +138,48 @@
 (def ^:private relative-date-string-decoders
   [{:parser #(= % "today")
     :range  (fn [_ dt]
-              {:start dt,
-               :end   dt})
+              (let [dt-res (.toLocalDate ^LocalDateTime dt)]
```

Comment:
You can just use `(t/local-date ...)` here to convert stuff to `LocalDate`s. Then you won't need to type hint everything

Reply:
Done